Formula/minio.rb

```diff
@@ -21,12 +19,24 @@ def install
     clipath = buildpath/"src/github.com/minio/minio"
     clipath.install Dir["*"]
 
-    cd buildpath/"src/github.com/minio/minio/" do
-      system "go", "build", "-o", bin/"minio", "-v"
+    cd clipath do
+      if build.head?
+        system "go", "build", "-o", buildpath/"minio"
+      else
+        minio_release = `git tag --points-at HEAD`
+        version = minio_release.gsub(/RELEASE\./, "")
+        minio_version = version
+        minio_commit = `git rev-parse HEAD`
+
+        system "go", "build", "-ldflags", "-X main.minioVersion=#{minio_version} -X main.minioReleaseTag=#{minio_release} -X main.minioCommitID=#{minio_commit}", "-o", buildpath/"minio"
+      end
     end
+
+    bin.install buildpath/"minio"
   end
 
   test do
-    system "#{bin}/minio"
+    minio_version = DateTime.parse(version).strftime("%Y-%m-%dT%H-%M-%SZ")
+    assert_match minio_version, shell_output("#{bin}/minio version", "version doesn't match")
```

Comment:
Second argument for `shell_output` should be left blank: https://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/5982/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_minio/